leap_ec/probe.py

```diff
@@ -206,6 +208,22 @@ def __call__(self, next_individual):
             yield ind
 
 
+def _to_quoted_list(arr):
+    return f"{json.dumps(arr.tolist())}"
+
+@contextlib.contextmanager
+def _maybe_json(numpy_as_json):
+    try:
```

Comment:
Similarly add an explanation as to why sometimes we want arrays as JSON vs. python lists would be helpful.  In particularly, the _genomes_ are numpy arrays, and ultimately the probes write those in CSV format.